src/sidebar/util/oauth-client.js

```diff
@@ -245,6 +212,41 @@ class OAuthClient {
     };
     return this.$http.post(url, data, requestConfig);
   }
+
+  /**
+   * Create a pop-up window for use with `OAuthClient#authorize`.
+   *
+   * This function _must_ be called in the same turn of the event loop as the
+   * button or link which initiates login to avoid triggering the popup blocker
+   * in certain browsers. See https://github.com/hypothesis/client/issues/534
+   * and https://github.com/hypothesis/client/issues/535.
+   *
+   * @param {Window} $window - The parent of the popup window.
```

Comment:
Perhaps this comment would be clearer if it indicated that the popup window is what is being created (and returned) by this method (e.g. `$window - The parent context of the popup window that will be generated and returned`). I found myself initially thinking "what popup window?...oh, the one this method creates. I see"

Reply:
Agreed. Will do.